package.json

```diff
@@ -26,8 +26,27 @@
     "test": "mocha $MOCHA_FLAGS"
   },
   "files": [
-    "lib/",
-    "src/",
```

Comment:
I want to make sure that `.*.sw?` files and the like are never packaged.  I'm not entirely sure what the concern is here